src/main/java/org/terasology/climateConditions/BodyTemperatureComponent.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2020 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
```

Comment:
I think @keturn switched the default template for the license header to a way shorter format - I don't know when/how IntelliJ picks this up, though, or where it is configured 🤔

Reply:
Yep, that PR got merged a couple weeks ago. Make sure your Terasology repo is up-to-date with `develop`, and if it the default doesn't change automatically, see Settings / Editor / Copyright and set the default copyright profile to "Terasology Foundation."